docs/getting-started/publish-infrastructure.md

```diff
@@ -0,0 +1,560 @@
+---
+title: Publish Infrastructure
+toc: true
+weight: 4
+indent: true
+---
+
+# Publish Infrastructure
+
+Provisioning infrastructure using the Kubernetes API is a powerful capability,
+but combining primitive infrastructure resources into a single unit and
+publishing them to be provisioned by developers and consumed by applications
+greatly enhances this functionality.
+
+As mentioned in the [last section], CRDs that represent infrastructure resources
+on a provider are installed at the *cluster scope*. However, applications are
+typically provisioned at the *namespace scope* using Kubernetes primitives such
+as `Pod` or `Deployment`. To make infrastructure resources available to be
+provisioned at the namespace scope, they can be *published*. This consists of
+creating the following resources:
+
+- `InfrastructureDefinition`: defines a new kind of composite resource
+- `InfrastructurePublication`: makes an `InfrastructureDefinition` available at
+  the namespace scope
+- `ClusterRole`: allows Crossplane to manage this newly defined infrastructure
+  type
```

Comment:
I think we can gloss over (i.e. omit) this one here, since it's not a Crossplane concept.

Reply:
Will address this depending on final resolution of https://github.com/crossplane/crossplane/pull/1514#discussion_r426226676